tools/MVP/template.py

```diff
@@ -89,17 +101,21 @@ def _generate_cpp_files(name: str, include_setup: bool, output_directory: str) -
         _generate_setup_file(name, _cpp_filename, "main", "cpp", output_directory)
         _generate_setup_file(name, _cpp_filename, "CMakeLists", "txt", output_directory)
 
+    if add_tests:
+        _generate_test_file(name, _cpp_filename, "Presenter", "h", output_directory)
+        _generate_test_file(name, _cpp_filename, "Model", "h", output_directory)
+
     print(f"Output directory: {output_directory}")
     print("Done!")
 
 
-def _generate_files(name: str, language: str, include_setup: bool, output_directory: str) -> None:
+def _generate_files(name: str, language: str, include_setup: bool, output_directory: str, add_tests: bool) -> None:
```

Comment:
Can the `add_tests` arguments in this file be updated to be `include_tests` to keep it consistent with the new naming? It might also be worth moving this argument to go before the `output_directory` argument in the function calls so that all these similiar options stay together for readability